examples/data-sources/grafana_slos/data-source.tf

```diff
@@ -0,0 +1,41 @@
+resource "grafana_slo" "test" {
```

Comment:
What's the difference between this example and the resource example below? This is just my ignorance showing.

Reply:
The `data-source.tf` file first creates a `grafana_slo` resource named `test`
on line 41, `data "grafana_slos" "slos" {}` we then perform a READ request on the datasource

it's essentially an example of how to populate a datasource, and then do a datasource READ.